src/components/Search/index.js

```diff
@@ -1,89 +1,88 @@
-      } = this.state;
-
-      return (
-        <div>
-          <StepNavigator stepNumber={2} stepLabel="Choose your job/school address" prevPath="/" />
-
-          <PlacesAutocomplete
-            value={this.state.address}
-            onChange={this.handleChange}
-            onSelect={this.handleSelect}
-            shouldFetchSuggestions={address.length > 3}
-          >
-            {({ getInputProps, suggestions, getSuggestionItemProps }) => (
-              <div className="search-bar-container">
-                <div className="search-input-container">
-                  <input
-                    {...getInputProps({
-                                        placeholder: 'Search Places ...',
-                                        className: 'location-search-input',
-                                    })}
-                  />
-                </div>
-                {suggestions.length > 0 && (
-                <div className="autocomplete-container">
-                  {suggestions.map((suggestion) => {
-                                        const className = suggestion.active ? 'suggestion-item--active' : 'suggestion-item';
-                                        const parts = suggestion.description.split(',');
-                                        if (parts.includes(this.state.city)) {
-                                            return (
-                                              <div
-                                    {...getSuggestionItemProps(suggestion, { className })}
-                                              >
-                                                <span><b>{parts[0]}</b></span>
-                                                <span>
-                                            ,{suggestion.formattedSuggestion.secondaryText}
-                                                </span>
+      });
+  };
 
-                                              </div>
-                                            );
-                                        }
-                                    })}
+  drawInputField = ({getInputProps, suggestions, getSuggestionItemProps}) => (
+    <div className="search-bar-container">
+      <div className="search-input-container">
+        <input
+          {...getInputProps({
+            placeholder: 'Search Places ...',
+            className: 'location-search-input',
+          })}
+        />
+      </div>
+      {suggestions.length > 0 && (
+        <div className="autocomplete-container">
+          {suggestions.map((suggestion) => {
+            const className = suggestion.active ? 'suggestion-item--active' : 'suggestion-item';
+            const parts = suggestion.description.split(',').map(part => part.trim());
+            if (parts.includes(this.props.city.name)) {
+              return (
+                <div
+                  {...getSuggestionItemProps(suggestion, { className })}
+                >
+                  <span><b>{parts[0]}</b></span>
```

Comment:
Why do we bold first part ? If it's cityName then why do we check in line 49 for all parts if includes if we could check only first element :)

Reply:
First element depends on input. It could be a road, city or even country. For example:
["Bus station", "Aleje" , "Warsaw", "Poland"]  // ["Chopina", "Warsaw"]. I bold the most detailed part = first one. I could bold city name if you prefer.